parm/prep/aero_emissions.yaml

```diff
@@ -0,0 +1,27 @@
+aero_emissions:
+  config:
+    debug: False
+    ratio: 0.95                                    # weighting ratio
+    emistype: 'QFED'                               # EMission Type, Valid answers: 'QFED, GBBEPx, HFED'
+    climfile_str: 'GBBEPx-all01GRID_v4r0_climMean' # climate file base string used for glob later
+    GBBEPx_version: 'v4r0'                         # gbbepx version
+    qfed_version: '006'                            # qfed version
+    species: [ 'so2','oc','bc' ]                   # species to be used
+    historical: False                              # set to true to just use true data for the given day
+    coarsen_scale: 150                             # scale for coarsen function to generate weights
+    output_vars: ["BC", "CH4", "CO", "CO2", "NH3", "NOx", "OC", "PM2.5", "SO2"]
+    input_vars: ["bc", "ch4", "co", "co2", "nh3", "no", "oc", "pm25", "so2"]
+    data_in:
+      mkdir:
+        - "{{ DATA }}"
+        - "{{ COMOUT_CHEM_HISTORY}}"
+      copy:
+        - ["{{ AERO_INPUTS_DIR }}/nexus/QFED/{{ current_cycle | strftime('%Y/%m') }}/qfed2.emis_oc.006.{{ current_cycle | to_YMD }}.nc4", "{{ DATA }}/"]
+        - ["{{ AERO_INPUTS_DIR }}/nexus/QFED/{{ current_cycle | strftime('%Y/%m') }}/qfed2.emis_so2.006.{{ current_cycle | to_YMD }}.nc4", "{{ DATA }}/"]
+        - ["{{ AERO_INPUTS_DIR }}/nexus/QFED/{{ current_cycle | strftime('%Y/%m') }}/qfed2.emis_bc.006.{{ current_cycle | to_YMD }}.nc4", "{{ DATA }}/"]
+        {% for fdate in forecast_dates %}
+        - ["{{ AERO_INPUTS_DIR }}/nexus/GBBEPx/v4/climMean/GBBEPx-all01GRID_v4r0_climMean_{{ fdate | strftime('%m%d') }}.nc", "{{ DATA }}/"] # copy climo files
```

Comment:
What happens when `emistype` is not `QFED`? Are these still the relevant files to be copied?

Reply:
if the emission type changes then we need to modify these lines.  I couldn't figure out how to put a condition in for the different emission types.  If you have a suggestion on how we can do this in jinja I'm happy to add it.  it will make everyone's lives easier.

Reply:
One idea, that is just simple, is:
```yaml
data_in:
    qfed:
         copy:
            -
            -
    hfed:
        copy:
            -
            -
```
Handle the `mkdir` part separately from `copy` since that is common.
Just a thought
You get everything right in front in the yaml

Reply:
I'll give it a shot